packages/core/src/modules/attendance/attendance-service.ts

```diff
@@ -109,10 +116,23 @@ export class AttendanceServiceImpl implements AttendanceService {
   async updatePool(poolId: AttendancePoolId, data: Partial<AttendancePoolWrite>) {
     const pool = await this.attendanceRepository.getPoolById(poolId)
 
+    // FIXME: refactor into own method for changing capacity to make this easier to manage.
     if (data.capacity && pool.numAttendees > data.capacity) {
       throw new AttendanceValidationError("Cannot change pool capacity to less than the reserved spots")
     }
 
+    if (data?.capacity && data.capacity > pool.capacity) {
+      const attendees = await this.attendeeService.getByAttendancePoolId(poolId) // These are in order of reserveTime
+
+      const attendeesThatAreNowReserved = attendees.slice(pool.capacity, data.capacity)
+      for (const attendee of attendeesThatAreNowReserved) {
+        const result = await this.attendeeService.tryReserve(attendee.id, pool)
+        if (result === false) {
```

Comment:
nit: !result

or can this also be null or undefined?

Reply:
Typen sa false eller attendee. Burde dobbeltsjekke